TCAT/Controllers/FavoritesViewController.swift

```diff
@@ -40,13 +63,14 @@ class FavoritesViewController: UIViewController {
     }
 
     private func setupButtons() {
-        let favoritesBlueColor = UIColor(hex: "08A0E0")
+        let editString = isEditingFavorites ? "Done" : "Edit"
         let attributes: [NSAttributedString.Key: Any] = [
             .font: UIFont.getFont(.regular, size: 14.0),
-            .foregroundColor: favoritesBlueColor,
+            .foregroundColor: Colors.notificationBlue,
         ]
```

Comment:
We could do this without attributed strings if we wanted. Idk which is better.

    editButton.setTitleColor(...)
    editButton.titleLabel?.font = ...

Reply:
I think not having the attributed string is better! Thanks!